design-proposals/klip-15-new-api-and-client.md

```diff
@@ -89,8 +93,6 @@ operation (newlines have been added here for the sake of clarity but the real JS
 ````
```

Comment:
Not part of the review changes, but above:
> (newlines have been added here for the sake of clarity but the real JSON must not contain newlines)

we might want to clarify "real JSON must not contain *unescaped* newlines"

Reply:
ack